webapp/src/ts/services/integration-api.service.ts

```diff
@@ -40,7 +40,7 @@ export class IntegrationApiService {
     this.AndroidAppLauncher = androidAppLauncherService;
     this.Language = languageService;
     this.Select2Search = select2SearchService;
-    this.Enketo = enketoService;
+    this.Enketo = enketoFormService;
```

Comment:
AFAICT, this is only used by the db-object-widget.  That widget calls the `getCurrentForm` function to load the current form in context.